src/Apps/W1/Subscription Billing/App/Service Commitments/Tables/SubscriptionLineArchive.Table.al

```diff
@@ -267,6 +267,15 @@ table 8073 "Subscription Line Archive"
         {
             Caption = 'Perform Update On';
         }
+        field(60; "Variant Code"; Code[10])
+        {
+            Caption = 'Variant Code';
+            ToolTip = 'Specifies the Variant Code of the Subscription.';
+            FieldClass = FlowField;
+            CalcFormula = lookup("Subscription Header"."Variant Code" where("No." = field("Subscription Header No.")));
+            Editable = false;
+            AutoFormatType = 0;
```

Comment:
This AutoFormatType looks strange for a code field.
Do we really need this now? I thought that only decimal fields require this now.

Besides that.
What is the different between this new FlowField and the already existing field `"Variant Code (Sub. Header)"`?